higlass/client.py

```diff
@@ -360,6 +420,26 @@ def add_location_sync(self, views_to_sync):
         # TODO: check that view already exists in viewconf
         self._add_sync("locationLocks", lock_id, [v.uid for v in views_to_sync])
 
+    def add_value_scale_sync(self, tracks_to_sync):
+        locks_map = self.conf["valueScaleLocks"]["locksByViewUid"]
+        locks_dict = self.conf["valueScaleLocks"]["locksDict"]
+        lock_id = slugid.nice()
+
+        for definition in tracks_to_sync:
+            v_uid, t_uid = self._extract_view_track_uids(definition)
+
+            if v_uid is None:
+                # If no view UID is found the definition seems to be broken
+                continue
```

Comment:
We should maybe display some sort of warning?

Reply:
Mhhh good question. How would you display it? A print statement would be bad because it would be displayed before the widget. I am wondering in general how jupyter widgets could best realize logging.

Reply:
I'm not sure I have a good answer, but a short warning either before or after the widget should be fine. As long as it isn't recurring it wouldn't be too much of a hassle.

Reply:
Warnings are added. To see them set `%env HIGLASS_PYTHON_DEBUG=True` and look at `higlass-python.log` next to your `.ipynb` file.